conformance_tests/sysman/test_sysman_driver/README.md

```diff
@@ -0,0 +1,18 @@
+# testSysmanDriver
+
+## Description
+
+This test suite is for validating driver Extension APIs provided in Sysman. Ensure that the test executable is run with
+root user permissions otherwise test will fail.
+
+### zesDriverGetExtensionProperties
+
+* `GivenValidDriverHandleWhileRetrievingExtensionPropertiesThenValidExtensionPropertiesIsReturned`:
```

Comment:
Please share whether there is a plan to maintain this readme for every test that is going to be added.

Reply:
Not sure. Since there are only two tests, i added it. In some other test suites like device, it was added in this format.